maintenance/annotation_maint.py

```diff
@@ -0,0 +1,38 @@
+import allel
```

Comment:
I tried to run this locally but got an error

```
~/projects/stdpopsim/maintenance annotation ?4 ❯ python3 annotation_maint.py                                                                                           stdpopsim-dev 13:25:30
Traceback (most recent call last):
  File "annotation_maint.py", line 21, in <module>
    x, y = urllib.request.urlretrieve(address, tmp_path)
  File "/opt/anaconda3/envs/stdpopsim-dev/lib/python3.7/urllib/request.py", line 288, in urlretrieve
    % (read, size), result)
urllib.error.ContentTooShortError: <urlopen error retrieval incomplete: got only 29639876 out of 37903989 bytes>

~/projects/stdpopsim/maintenance annotation ?5 ❯ python3 annotation_maint.py                                                                                   1m 48s  stdpopsim-dev 13:27:23
Traceback (most recent call last):
  File "annotation_maint.py", line 13, in <module>
    os.mkdir(annot_path)
FileExistsError: [Errno 17] File exists: 'annotations'
```

Reply:
looks like there was a downloading error?